### PR TITLE
[DFFA-4] Add Page Navigation to your Pages 

### PR DESCRIPTION
**Pull request by Aarinolaa**
I am not sure what happened to my files, i could not find them, seems they were deleted. So I created a new request. Thanks

image
image

### CODE REVIEW

**Review by adaugochi — CHANGES_REQUESTED** [review]
You are to create a link between your already existing page. Such that if I am on the product page and I click on `contact us`, it takes me to contact page and vice versa. You are not expected to create a new file

**Comment by Aarinolaa** [comment]
Hi Ada, like I had explained to Ireti earlier, I couldn’t access the
previous file hence my creating a new. Please advise if you can view the
previous one. Thank you

**Review by adaugochi — COMMENTED** [review]
image
image

Both files are already existing. If the file is not visible on your local, it means you haven't pulled from upstream yet to update those changes.

Kindly checkout to `master` and pull from `upstream master`.
Before pulling from upstream, check if you have set `upstream` by running `git remote -v`. If you see 4 links (i.e 2 `origin` and 2 `upstream`) it means you have set `upstream` else run this command `git remote add upstream https://github.com/Dufuna-CodeCamp/Dufuna-Fem` to set `upstream`

**Comment by Aarinolaa** [comment]
i have updated the file as requested, kindly review. Thanks

**Comment by adaugochi** [comment]
> i have updated the file as requested, kindly review. Thanks

You still have done what you are told

**Comment by Aarinolaa** [comment]
Hi Ada, I don’t understand what you mean please.

**Review by Ireti203 — CHANGES_REQUESTED** [review]
You don't need to create a navigationpage.html file. What you should do is to add the links to the previously existing files i.e on the product page file, add the link to the contact page and product page, do same on the contact page too. Also, remove the two images in the HTML-FORMS folder, they have no use there.

**Comment by Aarinolaa** [comment]
Hi Ireti,
Please is there someone I can speak with? Because I am not certain this
mail exchange is working for me. I need to hear someone tell me exactly how
to achieve this as it seems I am not understanding it.
I don’t have the previous files because my pc crashed and i lost my
documents hence my re-doing the work.
I need your help please.
Regards,

File: submissions/arinolaOsagie/HTML-FORMS/contact-form.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 </head>
 <body>
     <nav>
-        <a href="http://localhost:63342/dufuna-html/ProductPage.html?_ijt=20eggs136c3k6r8671nrtog35b">Product</a> <a href="http://localhost:63342/dufuna-html/HTML%20FORM.html?_ijt=6p04hq2tlbksa86k5jsajp3b12">Contact Form</a>
+        <a href="productpagehtml.html">Product</a> <a href="contactpage.html">Contact Form</a>
```
Comment (Ireti203) [inline submissions/arinolaOsagie/HTML-FORMS/contact-form.html:9]: I am very sure these links don't redirect when you click them. You should have contact-form.html and not contactpage.html i.e the name of the file. For the product page also, ensure you are using the exact name of the file. 

The links should be in both the product page and contact-form files

**Review by Ireti203 — CHANGES_REQUESTED** [review]
I thought you earlier pushed the images. Why are you pushing it again?

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**